src/breadcrumbs/breadcrumbs.scss

```diff
@@ -18,47 +18,67 @@
   user-select: none;
 }
 
-@mixin iui-breadcrumbs-item {
+@mixin iui-breadcrumbs-item($breadcrumbs-height: $iui-component-height) {
+  --_iui-breadcrumbs-item-separator-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z' /></svg>");
+
   display: flex;
-  line-height: $iui-component-height;
-  height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
+  align-items: center;
+  line-height: $breadcrumbs-height;
+  height: $breadcrumbs-height;
 
-  a {
-    @include iui-anchor;
+  &:not(:last-child)::after {
+    flex-shrink: 0;
+    width: $iui-icons-small;
+    height: $iui-icons-small;
+    margin: 0 $iui-xxs;
+
+    // #region IE support & future support
+    content: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z' fill='%23000' fill-opacity='0.4' /%3E%3C/svg%3E");
+    line-height: 14px;
+
+    @supports (mask: var(--_iui-breadcrumbs-item-separator-svg)) {
+      content: '';
+      background-color: var(--iui-icons-color);
+      mask: var(--_iui-breadcrumbs-item-separator-svg);
+    }
+    // #endregion IE support & future support
+
+    @media (forced-colors: active) {
+      background-color: CanvasText;
+    }
   }
+}
 
-  .iui-button {
-    @include iui-button-borderless;
-    margin: 0 (-$iui-s - 1);
+@mixin iui-breadcrumbs-text {
+  margin: 0 ($iui-s + 1);
+  max-width: 24ch;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+
+  @at-root a#{&} {
+    @include iui-anchor;
   }
+}
+
+@mixin iui-breadcrumbs-button {
+  @include iui-button;
+  @include iui-button-borderless;
 
-  > *,
   .iui-button-label {
```

Comment:
so this part is weird.. We kinda create a new `component` for breadrcums button, but inside still use same elements from original button..
And in react it makes no fun for these custom classes to be applied (without creating new components).
So I am not so sure anymore, what should be the best approach.. Including mixins and having new components, or just use existing components, especially, when there is no rules to be changed/overriden

Reply:
When we refactor the buttons the mixin should then reference `&-label` so we could use `.iui-breadcrumbs-button-label` instead of `.iui-button-label`.